squid/customize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# This file provides a parametrized version of
```

Comment:
```suggestion
# This file provides a parameterized version of
```

Reply:
According to Wikipedia, parametrized is the right spelling:
https://en.wikipedia.org/wiki/Parametrization

Reply:
Huh, look at that, works for me.